eng/common/pipelines/templates/steps/get-code-owners.yml

```diff
@@ -0,0 +1,24 @@
+parameters: 
+  TargetUserVariable: ''
+  TargetTeamVariable: 'notspecified'
+  TargetLabelVariable: 'notspecified'
+  ServiceDirectory: ''
+  DevOpsFeed: "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
+variables:
+    - template: ../variables/tool-version.yml
+steps:
+  - pwsh: |
+      dotnet tool install --tool-path . --add-source "${{ parameters.DevOpsFeed }}" --version '$("Azure.Sdk.Tools.RetrieveCodeOwners")' "Azure.Sdk.Tools.RetrieveCodeOwners"
```

Comment:
```
'$("Azure.Sdk.Tools.RetrieveCodeOwners")'
```

What is this?

Reply:
Perhaps `RetrieveCodeOwnersToolVersion` would simplify this a little.

Reply:
Yes I don't know if that quoting will actually work. Also based on the fact I don't believe you can specify variables inside of a step template we might need to pass the version as a template parameter for now.

Reply:
Here shows how I used it after testing the usage: 
https://github.com/Azure/azure-sdk-for-python/blob/add_author/eng/pipelines/templates/stages/archetype-python-release.yml#L271